docs/dev-env-setup.md

```diff
@@ -185,7 +191,13 @@ or
 npm run start
 ```
 
-You should now have a local instance of the site running at `http://localhost:5000/`. Please [file bugs](https://github.com/webcompat/webcompat.com/issues/new) if something went wrong!
+You should now have a local instance of the site running at `http://localhost:5000/`.
+
+#### Getting error messages?
+
+First you should have a look at the logs. They are located at `/tmp/webcompat.log`.
```

Comment:
or ping on IRC or slack https://webcompat.com/contact

Reply:
The log location is defined over here:

Reply:
this comment probably makes most sense after "Please file bugs"